CONTRIBUTING.md

```diff
@@ -10,14 +10,36 @@ To discuss anything please [contact Cornelius](mailto:cschum@suse.de).
 
 ## Contribute code
 
-To contribute code please open pull requests.
+To contribute code fork the repository and open pull requests with your changes.
 
 Ensure that rspec and rubocop pass locally before sending your PR and always that you add new changes.
 
 If your changes include important new features or bug fixes please add them to the [Master (unreleased) section fo the CHANGELOG.md](https://github.com/openSUSE/trollolo/blob/master/CHANGELOG.md#master-unreleased)
 
+### Set up
+
+Clone git repo: 
+
+`git clone git@github.com:openSUSE/trollolo.git`
+
+Build the gem: 
```

Comment:
Building the gem and the rest things you have added here, are not really to set the project up but to try it. As you could make changes and a PR without building the gem if the changes are small.

What about a _### To try your changes_ section after the _### To run rspec test_ one? 😄